pylint/checkers/base.py

```diff
@@ -276,7 +276,7 @@ def in_nested_list(nested_list, obj):
 
 def _get_break_loop_node(break_node):
```

Comment:
Can the code that uses this be changed to use `checkers.utils.get_node_first_ancestor_of_type`? 
This function is basically doing the same I believe.